lib/orderbook_sync.js

```diff
@@ -71,7 +76,7 @@ class OrderbookSync extends WebsocketClient {
       this.books[productID].state(data);
 
       this._sequences[productID] = data.sequence;
-      this._queues[productID].forEach(this.processMessage);
+      this._queues[productID].forEach(this.processMessage.bind(this));
```

Comment:
Good catch. The context can also be passed as an additional argument to `forEach`, so we'd end up with `this._queues[productID].forEach(this.processMessage, this);`.